core/dbt/adapters/factory.py

```diff
@@ -140,8 +140,6 @@ def get_adapter_plugins(self, name: Optional[str]) -> List[AdapterPlugin]:
                 raise InternalException(f"No plugin found for {plugin_name}") from None
             plugins.append(plugin)
             seen.add(plugin_name)
-            if plugin.dependencies is None:
```

Comment:
@VersusFacit This works right?  From what [I can see here](https://github.com/dbt-labs/dbt-core/blob/4e57c51c7abc5dd0134e14a3df27842c5fd57b65/core/dbt/adapters/base/plugin.py#L39) `plugin.dependencies` will never be `None`.